sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/analysis/AnalysisSuite.scala

```diff
@@ -62,7 +62,7 @@ class AnalysisSuite extends AnalysisTest with Matchers {
         a.select(UnresolvedStar(None)).select($"a").union(b.select(UnresolvedStar(None)))
       }
 
-    assertAnalysisSuccess(plan)
+    assertAnalysisSuccess(plan, maxIterations = Some(150))
```

Comment:
NOTE: We need more iterations here now because this PR adds `unionOutput` in `Union` then the analyzer needs to resolve output references.